### ISSUE TITLE
Double colon ":" in aligned text

### ISSUE THREAD

**Issue by jonasstein** [getnikola/nikola#3628]
### Environment
**Python Version:**
Python 3.9.13

**Nikola Version:**
Nikola 8.2.2

**Operating System:**
Gentoo Linux

### Description:
Aligned text in `.rst` with the following source

```
Öffnungszeiten
==============


:Montag:     geschlossen 
:Dienstag:   geschlossen
:Mittwoch:   11:30 - 15:00 und 17:00 - 21:00
:Donnerstag: 11:30 - 15:00 und 17:00 - 21:00
:Freitag:    11:30 - 15:00 und 17:00 - 21:00
:Samstag:    11:30 - 21:00
:Sonntag:    12:30 - 21:00
```
is rendered as: 
![image](https://user-images.githubusercontent.com/298682/172051090-d484f41f-22eb-42cc-b729-9f62ab439903.png)

I marked the section with the mouse. Note that one colon is not marked. Perhaps it is added by css?
See live website on http://yenyen.de/

**Comment by ralsina** [comment]
Yeah, it's coming from rst_base.css ... you can override it in your
custom.css but we should probably remove it, I'll do a quick PR for it.

**Comment by ralsina** [comment]
https://github.com/getnikola/nikola/pull/3629
